compiler/java/src/main/java/com/c/lexer/Token.java

```diff
@@ -0,0 +1,31 @@
+package com.c.lexer;
+
+public class Token {
+    private TokenType type;
+    private String value;
+    private int lineNum;
+    private int colNum;
+
+    public Token(TokenType type, String value, int lineNum, int colNum) {
```

Comment:
How do you handle tokens like `OR`? What do you pass as value? In the C++ implementation I have two constructors. One with type and value and one only with the type. Thats more efficient in my opinion

Reply:
The classes concerning the lexer are just dummies right now. Will work on them in the next branch. Keep this comment in mind for the next pull request ;)